src/main/java/tasks/Deadline.java

```diff
@@ -0,0 +1,28 @@
+package tasks;
```

Comment:
The classes in `tasks` perhaps have magic strings, following here: https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers